bindings/python/mpi_comm_shim.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * ghex-org
+ *
+ * Copyright (c) 2014-2023, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#pragma once
+
+#include <mpi.h>
+#include <pybind11/pybind11.h>
+#include <util/to_string.hpp>
+
+namespace pyghex
+{
+
+// https://github.com/arbor-sim/arbor/commit/1d6a48d0ce4b96f59acf931efd61d55c571c5e68
```

Comment:
Why a reference to arbor?

Reply:
taken from Arbor, will add clarification